samples/NativeLibrary/c_source/loadlibrary.c

```diff
@@ -0,0 +1,113 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#ifdef _WIN32
+#include "windows.h"
+#else
+#include "dlfcn.h"
+#define __stdcall
+#endif
+#include "stdio.h"
+#ifdef _WIN32
+#define __symLoad GetProcAddress
+#else
+#define __symLoad dlsym
+#endif
+#ifdef _WIN32
+#define _LibClose FreeLibrary
```

Comment:
Could you please use same style for `__symLoad` and `_LibClose` ? (Same number of underscores and same capitalization.)